src/styles.css

```diff
@@ -219,6 +242,9 @@ button#metrics-notification-warning-close {
   z-index: 99980;
   cursor: pointer;
   margin: 0 1em 0 0;
+  position: fixed;
+  bottom: 15px;
+  left: 15px;
```

Comment:
we should move the HTML for this toggle appropriately as well (to bottom of body)